### ISSUE TITLE
Add a way to Delete All Data

### ISSUE THREAD

**Issue by ddaddy** [AppTelemetry/Viewer#26]
During setup I am improving the data I send to get better readings on the platform.
I've ended up with a lot of `<None>` values where i've added some new values later.
Once I have everything as I like it, i'd like to clear the database completely.

I will likely create a new app on the platform, change the ID in my app, i'd then need to manually copy all my graphs/insights before deleting the old app.

Having a way to clear the database would be much easier.

I understand clearing the database is likely only going to be needed during app development.

**Comment by simonnickel** [comment]
With my previous Analytics solution I usually created a Test and Production analytics instance. This would be the alternative to deleting the database and allows to still try things out while the app already is live. But has the same downside of manually copying graphs/insights. Maybe a simple duplication of existing apps / configurations would be helpful?

**Comment by ddaddy** [comment]
Yes, either copy graphs/configurations to another app, or duplicate configurations to a new app (with a new ID).
Then i'm assuming if we simply delete an old app it also deletes all the data?

**Comment by winsmith** [comment]
By "all data" do you mean delete all signals but keep the insights? Would that be useful? Right now, my thinking is you just don't send signals in DEBUG mode and therefore your data does not get contaminated by development data.

**Comment by ddaddy** [comment]
Yeh basically start with no data as I publish a new app.
Going forward I'll use my PR with 2x ID's so debug data goes to its own app.
If I could copy the graphs from one to the other that would be great.

**Comment by winsmith** [comment]
Copying Insights sounds like a great idea! If I add a ticket for that, would that be enough or would you still like to keep this one open?

**Comment by winsmith** [comment]
Always super interesting to see how ppl use the system :D~

**Comment by ddaddy** [comment]
Sure copying insights would be all I need. Along with my PR to allow a DEBUG build to use Telemetry as we can use 2 different ID's and get a dashboard setup in debug app then copy the graphs to release app.

**Comment by winsmith** [comment]
I've created #28 but I think I'm keeping this one open as well, just a bit lower on the priority. 

Btw if you want you can ask me directly to delete all signals for a specific App ID by hand.
